packages/lesswrong/server/utils/feedUtil.ts

```diff
@@ -108,17 +116,31 @@ export function defineFeedResolver<CutoffType>({name, resolver, args, cutoffType
   });
 }
 
-export async function mergeFeedQueries<SortKeyType>({limit, cutoff, offset, subqueries}: {
+const applyCutoff = <SortKeyType>(
+  sortedResults: {sortKey: SortKeyType}[],
+  cutoff: SortKeyType,
+  sortDirection: SortDirection,
+) => {
+  const cutoffFilter = sortDirection === "asc"
+    ? ({sortKey}) => sortKey > cutoff
+    : ({sortKey}) => sortKey < cutoff;
+  return _.filter(sortedResults, cutoffFilter);
+}
+
+export async function mergeFeedQueries<SortKeyType>({limit, cutoff, offset, sortDirection, subqueries}: {
   limit: number
   cutoff?: SortKeyType,
   offset?: number,
-  subqueries: Array<any>
+  sortDirection?: SortDirection,
+  subqueries: Array<any>,
 }) {
+  sortDirection ??= "desc";
```

Comment:
Just learned a new bit of cursed javascript syntax

Reply:
It was even more cursed when we were still using Node 16 - Typescript and eslint would happily compile this, then Node would crash when it tried to parse it 😅